api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionSignPaths.scala

```diff
@@ -85,8 +93,58 @@ trait SubmissionSignPaths
 
     onComplete(fSubmission) {
       case Success(sub) =>
+        if (sub.status.code == 10) {
+          emailSignature(supervisor, sub)
+        }
         complete(ToResponseMarshallable(Receipt(sub.end, sub.receipt, sub.status)))
       case Failure(error) => completeWithInternalError(uri, error)
     }
   }
+
+  private def emailSignature(supervisor: ActorRef, submission: Submission)(implicit ec: ExecutionContext) = {
+    val emails = findEmailsById(submission.id.institutionId)
+    emails.map(emailSeq => {
+      emailSeq.foreach(t => {
+        val username = t._1 + " " + t._2
+        sendMail(t._3, username, submission)
+      })
+    })
+  }
+
+  private def sendMail(address: String, username: String, submission: Submission) = {
+    val config = ConfigFactory.load()
+    val host = config.getString("hmda.mail.host")
+    val port = config.getString("hmda.mail.port")
+
+    val properties = System.getProperties
+    properties.put("mail.smtp.host", host)
+    properties.put("mail.smtp.port", port)
+
+    val session = Session.getDefaultInstance(properties)
+    val message = new MimeMessage(session)
+
+    val date = getFormattedDate
+
+    val text = s"$username,\n\nCongratulations, you've completed filing your HMDA data for filing period ${submission.id.period}.\n" +
+      s"We received your filing on: $date\n" +
+      s"Your receipt is: ${submission.receipt}"
+    message.setFrom(new InternetAddress("no-reply@test.com"))
+    message.setRecipients(Message.RecipientType.TO, address)
+    message.setSubject("HMDA Filing Successful")
+    message.setText(text)
+
+    log.info(s"Sending message to $address with the message \n$text")
+    Transport.send(message)
```

Comment:
Is it possible to have a spec for this? It would be great to test that the message gets sent correctly and exactly once.

Reply:
Given some of the research I've done, I don't think it's currently possible in our unit tests.  Whenever we start automating integration tests, that's definitely something we can look into.